src/web/modules/gateway_accounts/detail.njk

```diff
@@ -75,6 +75,18 @@
             <td class="govuk-table__cell">{{ account.toggle_3ds | string | capitalize }}</td>
           </tr>
       {% endif %}
+      {% if account.allow_apple_pay != undefined %}
+          <tr class="govuk-table__row">
+            <th class="govuk-table__header" scope="col">Apple Pay Enabled</th>
```

Comment:
Minor comment but the design system uses "sentence case" so these should read "Apple Pay enabled" and "Google Pay enabled". 
https://design-system.service.gov.uk/styles/typography/#headings

Just making sure @jonheslop comments live on!

Reply:
Done and added '3DS enabled' instead of 'Toggle 3DS'